src/constants/labels.json

```diff
@@ -22,7 +22,7 @@
     },
     "qrCodeScanner": {
       "noAuthorization": {
-        "description": "Kamerazugriff zum Scannen von QR-Codes erforderlich. Öffnen Sie die Android-Einstellungen, tippen Sie auf Berechtigungen und tippen Sie auf Erlauben.",
+        "description": "Kamerazugriff zum Scannen von QR-Codes erforderlich. Öffnen Sie die Einstellungen, tippen Sie auf Berechtigungen und tippen Sie auf Erlauben.",
```

Comment:
I guess this only applies for android, not sure how you can grant permissions in iOS.

Reply:
i mean we could adjust these text but there is a `Einstellungen` Button below the text and then you reach the settings section on each platform

Reply:
Even though this is incorrect on iOS, it is fine for me. There is no such thing as `Berechtigungen` or `Erlauben`. You just toggle the switch `Kamera`. If you open the settings directly and not from the button in lunes, you just need to select `Lunes` and toggle the switch.